.github/workflows/build_and_deploy.yml

```diff
@@ -178,7 +178,7 @@ jobs:
         token: ${{ secrets.GIT_HUB_SERVICE_ACCOUNT_TOKEN }}
         checkName: Deploy ${{ env.DOCKER_IMAGE_TAG }} to ${{ env.ENVIRONMENT }} # Job name within triggered workflow
         ref: ${{ env.BUILD_GIT_TAG }}
-        timeoutSeconds: 720
+        timeoutSeconds: 840
```

Comment:
Could we bump it even higher to guarantee success? Like 1200 (20 min)

Reply:
Also could you fix the task id? Noticed it's still called "wait_for_deploy_app_staging"